### PR TITLE
XS✔ ◾ ADR improvements

### PR DESCRIPTION
**Pull request by bradystroud**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️

> 2. What was changed?

✏️

> 3. Did you do pair or mob programming (list names)?

✏️

### CODE REVIEW

File: rules/architectural-decision-records/rule.md [hunk 1]
```diff
@@ -1,86 +1,106 @@
...
+
+redirects:
+  - architectural-decision-records
+created: 2023-06-26T06:03:20.995Z
+guid: a131455c-96db-4c0d-829c-20a506c1bcc8
+---
+
+Architectural Decision Records (ADRs) are lightweight documents use to record important decisions in your project. They do not necessarily have to be related to architecture, but could be any important decision made by the team.
+
+### What are the dangers of not documenting important decisions?
```
Comment (danielmackay) [inline rules/architectural-decision-records/rule.md:23]: Why change from H2 to H3?

File: rules/architectural-decision-records/rule.md [hunk 2]
```diff
@@ -1,86 +1,106 @@
...
+
+1. Lack of transparency and communication
+2. Loss of intellectual property
+3. Loss of historical context
+4. Risk of repeating mistakes
+5. Difficulty in auditing and governance
+
+<!--endintro-->
```
Comment (danielmackay) [inline rules/architectural-decision-records/rule.md:31]: Why move the intro?  Seems weird to include a heading.

File: rules/architectural-decision-records/rule.md [hunk 3]
```diff
@@ -1,86 +1,106 @@
...
+
+### What are the advantages of using ADRs?
```
Comment (danielmackay) [inline rules/architectural-decision-records/rule.md:33]: Can you add a section on 'What is an ADR'?  You can copy from my slides.

**Review by danielmackay — CHANGES_REQUESTED** [review]
Few comments

File: rules/architectural-decision-records/rule.md (hunk 1 above)
Comment (bradystroud) [reply rules/architectural-decision-records/rule.md:23]: I know it doesnt make sense in terms of heading hierarchy, but H2s feel really big, happy to stick with them though.

File: rules/architectural-decision-records/rule.md (hunk 2 above)
Comment (bradystroud) [reply rules/architectural-decision-records/rule.md:31]: I think the dangers is good to be in the intro, but yes I agree its weird to have a heading.

I'll undo that

**Review by danielmackay — APPROVED** [review]
LGTM

**Pull request merged**